PortalForecasts.R

```diff
@@ -156,12 +156,13 @@ forecastall <- function(abundances,level,weather_data,weatherforecast) {
       model_aic = 1e6
     } else {
       model=tsglm(species_abundance,model=list(past_obs=1,past_mean=12),distr="nbinom")
```

Comment:
Should there still be 12's in the code? Or are they distinct from `num_forecast_months`?